2/first.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="2.css" />
```

Comment:
Já para não falar do nome do ficheiro!

A importação de um ficheiro deve ser sempre feita com `./etc...` se estive no mesmo diretório, ou com `../etc...` se tiveres de andar para trás nos diretórios:
```suggestion
    <link rel="stylesheet" href="./2.css" />
```